src/app/components/form/form.component.ts

```diff
@@ -233,10 +233,18 @@ export class FormComponent implements OnInit, OnChanges, OnDestroy {
     }
   }
 
+  // Remove trailing spaces and dots, and leading spaces
+  private sanitizeDomain(domain: string): string {
+    return domain.replace(/[ \.]+$/, '').trim();
+  }
+
   public runDomainCheck(submitValid = true) {
     this.form.markAllAsTouched();
     let param = this.form.value;
 
+    param.domain = this.sanitizeDomain(param.domain);
```

Comment:
The root `.` will be sanitized. Maybe prevent running the sanitize function if testing the root domain.

Reply:
Oh yes, that's right

Reply:
Should be better now